.github/workflows/cargo-doc.yaml

```diff
@@ -15,6 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: cachix/install-nix-action@v25
```

Comment:
If you want to make use of the new S3-backed cache for cosmos.nix, you can do the same as the Hermes and Cosmos.nix CI: https://github.com/informalsystems/cosmos.nix/blob/3fadaadd13838b487932aac31b8074dded74cf69/.github/workflows/ci.yml#L17-L21

Reply:
Good idea, let's work on this on a separate PR.